Library/Homebrew/dev-cmd/tests.rb

```diff
@@ -62,11 +62,12 @@ def run_buildpulse
 
     ohai "Sending test results to BuildPulse"
 
-    result = quiet_system Formula["buildpulse-test-reporter"].opt_bin/"buildpulse-test-reporter",
-                          "submit", "#{HOMEBREW_LIBRARY_PATH}/test/junit",
-                          "--account-id", ENV.fetch("HOMEBREW_BUILDPULSE_ACCOUNT_ID"),
-                          "--repository-id", ENV.fetch("HOMEBREW_BUILDPULSE_REPOSITORY_ID")
-    odie "Failed to send test results to BuildPulse!" unless result
+    system_command! Formula["buildpulse-test-reporter"].opt_bin/"buildpulse-test-reporter",
```

Comment:
```suggestion
    system_command Formula["buildpulse-test-reporter"].opt_bin/"buildpulse-test-reporter",
```

as discussed on Slack to ignore this error for now